dags/springer/dag_pull_ftp.py

```diff
@@ -19,12 +21,21 @@ def springer_pull_ftp():
     logger = get_logger().bind(class_name="springer_pull_ftp")
 
     @task()
-    def migrate_from_ftp(
-        sftp: SFTPService = SpringerSFTPService(),
-        repo: IRepository = SpringerRepository(),
-        **kwargs
-    ):
-        return pull_ftp.migrate_from_ftp(sftp, repo, logger, **kwargs)
+    def migrate_from_ftp(repo: IRepository = SpringerRepository(), **kwargs):
```

Comment:
I wouldn't change the signature of the function here, it will be inconsistent with the other publishers, we anyway passing the `sftp` instance, right?

Reply:
Actually, no, we don't pass it. It is put in the default args to not initialize in the func. Is incorrect,
Referring also to the [article](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/) you showed me once